components/PanelErrorBanner/PanelErrorBanner.tsx

```diff
@@ -37,9 +37,13 @@ const InnerWrapper = styled.div``;
 
 const ErrorMessage = styled.p`
   overflow-wrap: anywhere;
+  text-transform: capitalize;
```

Comment:
why are we capitalizing this? its just a regular sentence, not a title

Reply:
i actually just want to capiatlize the first letter of the first word because it makes the default ethers errors look better. i can remove this though and do it in the string formatting